### DISCUSSION TITLE
Version 0.9.4?

### DISCUSSION THREAD

**Discussion by CM000n** [DurgNomis-drol/mytoyota#247] [General]
Should we release another version 0.9.4 with the quick fix (https://github.com/DurgNomis-drol/mytoyota/pull/240) for the MyToyota changes or should we wait for the big bang release with the changeover to the new API (https://github.com/DurgNomis-drol/mytoyota/pull/242) @DurgNomis-drol & @joro75?

**Comment by joro75** [comment]
Hi @CM000n,

The quick fix for the MyToyota changes is already included in 0.9.3, which was recently released. 
As far as I know there is no reason for another release at this moment.

**Comment by DurgNomis-drol** [comment]
As @joro75 points out, it should be in 0.9.3

**Comment by CM000n** [comment]
Ah you are right. Thx for clarification.
